ext/drmgr/drmgr.h

```diff
@@ -756,6 +756,17 @@ bool
 drmgr_register_thread_init_event_ex(void (*func)(void *drcontext),
                                     drmgr_priority_t *priority);
 
+DR_EXPORT
+/**
+ * Registers a callback function for the thread initialization event,
+ * ordered by \p priority. Allows for the passing of user data \p usr_data
+ * which is available upon the execution of the callback. drmgr calls \p func
+ * whenever the application creates a new thread. \return whether successful.
+ */
+bool
+drmgr_register_thread_init_event_usr_data(void (*func)(void *drcontext, void *usr_data),
```

Comment:
Please add a line in api/docs/release.dox (at the bottom of the list [here](https://github.com/DynamoRIO/dynamorio/blob/master/api/docs/release.dox#L230))